### ISSUE TITLE
Setting an input_select option unknown to an open Dashboard causes it to set it back to the previous option.

### ISSUE THREAD

**Issue by DarkFox** [home-assistant/appdaemon#330]
**Steps to reproduce:**

- Create an input_select in HA with a number of items
- Add that input_select to a Dashboard
- Load the dashboard up in a browser
- Add a new option to the input_select
- Restart HA
- Select the new option in the input_select in the HA frontend

**Expected:**
The new option should be selected.

**Actual:**
The new option is briefly selected, then the value is switched back to the previously selected.

**What's happening:**
The Dashboard with the old list of options changes the value back.

**Workaround:**
Reload every open dashboard with the input_select, to make sure they have the updated list of options.

**Comment by ReneTode** [comment]
yeah if you have restarted HA, you should restart the dashboards.
also when you restart appdaemon.

Andrew has been working on it to update everything in appdaemon and the dashboards automaticly, but that is a very hard piece to do, because there are lots of things and environments to think about.

most things will work without reloading, but with the speed that HA is developing its hard to follow up on changes. so i advice reloading.

**Comment by acockburn** [comment]
As rene says, the workaround is to reload the dash when restarting hass - a specific fix for this would be a lot of work and isn;t worth the effort.
